### ISSUE TITLE
formatToCapitalized don't allow space-key?

### ISSUE THREAD

**Issue by guitexa** [VitorLuizC/brazilian-values#38]
It's me again 😁

I'm trying to use `formatToCapitalized` function, but it's impossible to type `space-key` as you can see in this example on SandBox below:

https://codesandbox.io/s/keen-bose-k6y63

Thanks for helping 🙏🏼

**Comment by VitorLuizC** [comment]
Hmmm, i'm currently using `String.prototype.trim` to prevent trailling whitespaces. So, no it don't.

I'll add an option to disable this behavior

**Comment by VitorLuizC** [comment]
Fixed and published under [version 0.8.0](https://github.com/VitorLuizC/brazilian-values/releases/tag/0.8.0).

Now you can disable trim trailing white-spaces with `trimTrailingWhiteSpaces: false`.

```js
formatToCapitalized(' com espaços antes e depois ', {
  trimTrailingWhiteSpaces: false
})
//=> ' Com Espaços Antes e Depois '
```

**Comment by guitexa** [comment]
Show man, it's worked fine, thanks! 🙏🏼
